rs/cli/src/commands/node_metrics/mod.rs

```diff
@@ -0,0 +1,42 @@
+use super::{ExecutableCommand, IcAdminRequirement};
+use clap::{Args, Subcommand};
+use from_node_metrics::FromNodeMetrics;
+use from_subnet_management::FromSubnetManagement;
+
+mod from_node_metrics;
+mod from_subnet_management;
+
+#[derive(Args, Debug)]
+pub struct NodeMetricsCmd {
+    #[clap(subcommand)]
+    pub subcommand: NodeMetricsCommand,
+}
+
+#[derive(Subcommand, Debug)]
+pub enum NodeMetricsCommand {
+    FromSubnetManagementCanister(FromSubnetManagement),
+    FromNodeMetricsCanister(FromNodeMetrics),
```

Comment:
These would translate to `from-subnet-management-canister` and `from-node-metrics-canister`, maybe they are too verbose? If they have a common pattern (`from-...-canister`) then maybe that part is not needed?
What do you think about
```
dre node-metrics from management-canister
dre node-metrics from metrics-canister
```

Seems a bit more fluent? If you want to do something like that you can follow the example in `version` command. Maybe even `-canister` is too much since I would assume that all targets are canisters. Also would add some aliases like `mgmt` for `management` 😄

Reply:
I agree it would be more fluid but maybe not as meaningful as having `-canister` in it. but probably we can do the from subcommand ;)